packages/shortcuts-extension/schema/plugin.json

```diff
@@ -157,6 +157,15 @@
       },
       "type": "object"
     },
+    "filemenu:persist-and-save": {
+      "default": { },
+      "properties": {
+        "command": { "default": "filemenu:persist-and-save" },
+        "keys": { "default": ["Accel Ctrl S"] },
```

Comment:
Whoops, should have noticed earlier: this keyboard shortcut is no good, since `Accel` *is* `Ctrl` on non-Mac platforms!

Reply:
Good catch. I switched it to `Ctrl Shift S`.